src/Thesis/Query.hs

```diff
@@ -0,0 +1,7 @@
+module Thesis.Query where
+
+import Thesis.Ast
+import Thesis.Types
+
+data Query = Query { getEpsilon :: Epsilon, getDelta :: Delta, ast :: Ast, defaultAnswer :: Double }
```

Comment:
`Epsilon` should be `Positive Epsilon`, and `Delta` should be `NonNegative Delta`